certificate-renewal/Certificate Renewal JSON contract.md

```diff
@@ -0,0 +1,195 @@
+      "reviewLine1",
+      "reviewLine2",
+      "reviewLine3",
+      "reviewLine4",
+      "reviewLine5",
+      "reviewLine6"
+    ]
+  }
+}
+```
+ - ***currentState:***
+  	 - 1-**name**: Share nickname of targeted certificate/club share. If no nickname, share description.
+  	 - 2-**maturityDate**: The current maturity date of the targeted certificate/club.
+  	 - 3-**minimumBalance**: The current minimum balance requirement of targeted certificate/club.
+  	 - 4-**currentStatus**: The current maturity option set for the targeted certificate/club.
+  	 - 5-**transferSLID**: If the current maturity option includes transferring funds into or out of the CD, this will be the current 10-digit account number and share/loan ID.
+  	 - 6-**transferAmount**: If the current maturity option includes transferring funds into or out of the CD, this will be the current amount to be transferred.
+
+  - ***options***: comma-delimited list of available options to member (1 thru 6). Options listed will be based upon the allowable options configured by the CU.			 
+	 - 1-**Increase certificate balance**:  Add additional funds to the certificate and renew.
+	 - 2-**Change certificate term**: Roll funds over into a new certificate type (requires CU contact).
+	 - 3-**Transfer certificate balance**: 
+		 - Full amount - all funds will be transferred out and certificate closed
+		 - Partial amount - limited to min balance requirement. Certificate will be renewed
+	 - 4-**Renew**:  Renew certificate to current terms.
+	 - 5-**Disburse funds by check**: Withdraw balance by check, close certificate.
+	 - 6-**Suspend**: Suspend certificate activity - funds retained in share.
+
+ - ***transferInOut***: A list of shares eligible for transferring certificate balance into or out of shares and loans as a source of funds for increasing certificate balance 
+	 - 1-**shareLoanId**: 4-character share or loan ID
+     - 2-**shareLoanName**: share nickname if available otherwise the share description
+	 - 3-**shareLoanBal**: the current share available balance
+
+ - ***payeeAddress***:  Comma-delimited member payee address lines representing the address the check will be mailed to should the member elect the 'Disburse funds by check' option. This is a  system calculated value containing from 1 to 6 Payee lines.
+
+ - ***payeeTerms***:  CU customizable terms to be displayed should the member elect the 'Disburse funds by check' option. Optional. May contain from 0 to 9 lines.
+
+ - ***suspendMessage***:  CU customizable message to be displayed should the member select the 'Suspend' option. May contain from 0 to 9 lines.
+ 
+ - ***reviewMessage***:  CU customizable message to be displayed on the review screen. Required. Must contain from 1 to 9 lines.
+
+## Request (PROCESSDATA):
+```json
+{
+  "rgState": "PROCESSDATA",
+  "powerOnFileName": "BANNO.CD.RENEW.V1.POW",
+  "userChrList":[
+    {"id": 1, "value": "0123456789S0123"}
+  ],
+  "userNumList": [
+    {"id": 1, "value": 1},
```

Comment:
@tkainz what's the reasoning behind using `userNumList` here? I don't know that we've ever used it before, and I don't necessarily object to using it, but just wondered why. Also, I thought this could only contain numbers for the value properties. Is the value on line 143 going to be a problem, since it's technically now a string (as far as JSON is concerned)?

Reply:
@jrobinson01 Good point! At first I included it as a numerical value because it seemed to make more sense as a number as  opposed to being handed off as a character variable. But then in looking back we've been passing any dollar values as characters so I'll bounce it up under the share/loan ID detail as userChrList - id 2. I'll hold off committing it in case there are any other changes also needing to be made.